bike_rent/views/bike_rent_views.xml

```diff
@@ -86,7 +87,7 @@
 
 <menuitem name="Data" id="menu_2" parent="menu_root"/>
 
-<menuitem id="bike_rent.menu_1_action" parent="menu_1" action="action_window_bike_rent"/>
+<menuitem id="menu_1_action" parent="menu_1" action="bike_rent_action"/>
```

Comment:
[For a menu: <model_name>_menu](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#security-view-and-action)